### ISSUE TITLE
Delete not working while the video is playing in the background

### ISSUE THREAD

**Issue by denz2** [EmbySupport/Emby.WMC.UI#30]
Thank you for fixing the delete but there is still a small issue if the video is playing you can't delete it.

**Comment by softworkz** [comment]
And why should this work?
I mean, when it's deleted it can't play, so you can't delete it while playing, doesn't it make sense?

**Comment by denz2** [comment]
It's just something that we get used to in WMC. It also doesn't make sense that you have to start to watch a new tv recording just to delete the recording that you just watched. If the remotes had a stop button it would be easier.

**Comment by LukePulverenti** [comment]
I don't remember anyone ever mentioning or asking for this now all of a sudden you're the second person to bring it up in the last few days. If it was there before it was mainly by accident given that it's the same context menu used everywhere else. It can be added back, but we have to actually make it work first, meaning the player and the UI have to handle it gracefully as you would expect. Right now I think things would just go haywire if we were to just put the menu option back in there and not do anything else.

**Comment by denz2** [comment]
Recorded tv is a different beast to anything else as it is constantly played and deleted and repeat many times a day it is a big downside if you are not able to delete quickly and easily. One family member the first thing they tried after I told them delete is working they go no it's not because they are so used to it. They are all back to using wmc on the pc's even with the crashes they said slow tv tunning and now difficulties with delete it's a thumbs down from them. 

Anyway this is the closest I got them to convert to emby, they use emby on all the other devices but not on htpc computers.

**Comment by softworkz** [comment]
So what is expected to happen when executing delete on a currently playing video?
Stop playing and then deleting?

**Comment by denz2** [comment]
Correct to stop it and remove it from the view and the in focus is the recording behind it.

**Comment by softworkz** [comment]
Yup, understood. It's one of the hundred little things which made WMC so good.
